scripts/report-governance-slos.py

```
@@ -363,11 +532,82 @@ def main() -> None:
+
+    # Data Collection Phase
+    if fixtures_mode:
+        fixtures_dir = safe_resolve_dir(base_dir, args.fixtures_dir)
+        if not fixtures_dir.exists():
+            fail(f"Fixtures directory not found: {fixtures_dir}")
+
+        data = collect_fixture_inputs(fixtures_dir)
+        mode = "fixture"
+        repository_name = repo or "fixtures/software-delivery-pipeline"
+    else:
+        if subprocess.run(["which", "gh"], capture_output=True).returncode != 0:
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)